x-pack/plugins/lens/public/editor_frame_service/editor_frame/editor_frame.test.tsx

```diff
@@ -7,6 +7,17 @@
 
 import React, { ReactElement } from 'react';
 import { ReactWrapper } from 'enzyme';
+
+// Mock the AutoSizer inside EuiSelectable (Chart Switch) and return some dimensions > 0
```

Comment:
Nit: Could you leave more detailed info why we have to do this (that the Autosizer behaves in particular way here)?

Reply:
Agreed with Marta- can you specifically say that Jest tests render using jsdom and [jsdom does not implement any sizing methods](https://github.com/jsdom/jsdom#unimplemented-parts-of-the-web-platform)